backend/baseclass.pm

```diff
@@ -908,13 +906,13 @@ sub _reduce_to_biggest_changes {
 
 sub freeze_vm {
     my ($self) = @_;
-    bmwqemu::diag "ignored freeze_vm";
+    bmwqemu::fcterr("ignored freeze_vm");
```

Comment:
For now i'd like to keep it as an error. As it's something not supported and a proper notice should be given to the test reviewer/backend developer